okhttp-logging-interceptor/src/test/java/okhttp3/logging/HttpLoggingInterceptorTest.kt

```diff
@@ -903,6 +903,101 @@ class HttpLoggingInterceptorTest {
       .assertNoMoreLogs()
   }
 
+  @Suppress("INVISIBLE_MEMBER", "INVISIBLE_REFERENCE")
```

Comment:
@MinhDang685 we run these tests in android also, which means they need to workaround internal methods. So I added this.